PureBasicIDE/ColorPicker.pb

```diff
@@ -1114,6 +1114,18 @@ Procedure ColorPicker_Name_Event(*Entry.ColorPickerData, Gadget, Type)
       EndIf
       
     Case #GADGET_Color_Canvas1
+      If Type = #PB_EventType_MouseWheel
+        *Entry\First - GetGadgetAttribute(#GADGET_Color_Canvas1 , #PB_Canvas_WheelDelta)
```

Comment:
That is a little bit too much with the spaces 😃 The convention for function parameters is to have one space after the comma, but none before it:
```
GetGadgetAttribute(#GADGET_Color_Canvas1, #PB_Canvas_WheelDelta)
```

Reply:
Yes I see that :-) , now I have fixed that and clicked commit here on Github, do I now have to submit a new pull request?